packages/@adobe/spectrum-css-temp/components/button/index.css

```diff
@@ -62,7 +62,8 @@ governing permissions and limitations under the License.
               box-shadow var(--spectrum-global-animation-duration-100) ease-out;
 
   text-decoration: none;
-  font-family: var(--spectrum-font-family-base);
+
+  @extend %i18nFontFamily;
```

Comment:
similar, this introduces a font-size and color, so we'll have to pay close attention to these components in particular when we run chromatic again
but it looks promising :)